CONTRIBUTING.md

```diff
@@ -38,11 +38,21 @@ We welcome pull requests!
 
 - edit code in the `./attack` directory
 - commit changes on branch
-- From the root of the repo, `cd attack && CONTAINER_TAG=super-cool-feature make docker-push` to push the tagged attack container
+- From the root of the repo, `cd attack && DOCKER_HUB_ORG=<dockerhub-user> CONTAINER_TAG=super-cool-feature make docker-push` to push the tagged attack container
 - `cd .. && make run` to run the launch container
-- `simulator infra create --attack-container-tag=super-cool-feature`
+- `simulator infra create --attack-container-tag=super-cool-feature --attack-container-repo=<dockerhub-user>/simulator-attack`
+
+### Implementation detailsj
```

Comment:
Grammar typo.

Reply:
Oops thats me trying to get out of insert mode in vim :fearful: